test/test_samples_osw/upgrades/results_output.csv

```diff
@@ -0,0 +1,11 @@
+project_national-0001.osw,1,0.0,0.0,5.5,54.4,0.0,0.0,45.5,780.2,0.0,0.0,156.9,0.0,222.1,94.2,0.0,0.0,59.8,0.0,0.0,0.0,0.0,1465.0,1327.7,0.0,0.0,0.0,0.0,-0.0,11.0,0.0,434.3,0.0,0.0,0.0,0.0,1138.0,1138.0,0.0,0.0,0.0,43.0,29.0,2262.4,3786.5,0.0,1.0,0.0,16.1,0.0,0.0,127.2,0.0,0.0,0.0,109.2,4656.5,41.1,2762.0,30.0,0.0,0.0,0.0,0.0,18.8,18.6,18.8,1.2,1.1,1.2,0.4,0.3,0.3,19.3,17.9,1.5,1.4,5468.0,1.7,0.0,7.7,0.0,12.9,30.0,4656.5,41.1,0.0,252.5,0.0,0.0,2762.0,Windows,201.2,201.2,0.0,60.4,0.0
+project_testing-0001.osw,0,40.0,0.0,16.4,0.0,0.0,0.0,0.0,405.1,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,43.3,-12466.4,11.0,0.0,0.0,0.0,159.2,0.0,2731.3,8194.0,0.0,0.0,0.0,0.0,0.0,65.0,27532.8,105902.4,0.0,1.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,-11831.5,1862.9,,,0.0,0.0,1903.2,0.0,15.4,21.0,16.9,3.7,0.1,0.2,0.0,0.0,0.0,15.3,16.5,4.8,0.1,4862.0,6.5,3053.7,94.0,0.0,361.4,0.0,634.9,1905.4,0.0,0.0,1903.3,0.0,0.0,Windows,5235.4,5614.8,1745.1,0.0,0.0
+project_national-0001.osw,1,0.0,0.0,5.5,54.4,0.0,0.0,45.5,1041.5,0.0,0.0,156.9,0.0,299.3,96.9,0.0,0.0,61.5,0.0,0.0,0.0,0.0,1465.0,1327.7,0.0,0.0,0.0,0.0,-0.0,11.0,0.0,434.3,0.0,0.0,0.0,0.0,1138.0,1138.0,0.0,0.0,0.0,414.0,32.0,2262.4,3786.5,0.0,1.0,0.0,16.1,0.0,0.0,130.8,0.0,0.0,0.0,109.1,4999.4,42.7,444.6,999.0,0.0,0.0,0.0,0.0,18.9,18.5,18.4,1.3,1.1,1.2,0.4,0.2,0.3,19.6,17.3,1.6,1.4,5468.0,1.7,0.0,7.7,0.0,12.9,30.0,4999.4,42.7,0.0,256.0,0.0,0.0,444.6,Walls,201.2,201.2,0.0,60.4,0.0
```

Comment:
Don't these need unique names so that diffs can be calculated?

Reply:
I updated the `OSW` column to look like `project_national-Walls.osw`.